src/libsync/configfile.cpp

```diff
@@ -896,10 +896,10 @@ void ConfigFile::setPromptDeleteFiles(bool promptDeleteFiles)
 bool ConfigFile::monoIcons() const
 {
     QSettings settings(configFile(), QSettings::IniFormat);
-    bool monoDefault = false; // On Mac we want bw by default
-#ifdef Q_OS_MAC
-    // OEM themes are not obliged to ship mono icons
-    monoDefault = (0 == (strcmp("ownCloud", APPLICATION_NAME)));
+    // On Mac and Linux we want bw by default
+    bool monoDefault = (0 == (strcmp("Nextcloud", APPLICATION_NAME)));
```

Comment:
Just a nitpick, could we use the opportunity to get rid of strcmp? I know that `QByteArrayLiteral("Nextcloud") == QByteArrayLiteral(APPLICATION_NAME)` is less performant but it's much more readable I think.